frontend/src/components/MealSection.js

```diff
@@ -1,23 +1,53 @@
 import MealItem from './MealItem'
 import dropdownIcon from '../assets/icons/dropdown.svg'
+import dropupIcon from '../assets/icons/dropup.svg'
+import React from 'react';
 
-const MealSection = (props) => {
+class MealSection extends React.Component {
 
-    return (
```

Comment:
We should only be using functional components - is there a reason why you switched to a class component?

Reply:
Lmk if i can help explain function react hooks - you'll want to use the `useState` hook here

Reply:
I used some online tutorial for getting the dropdowns to work, so the switch to the class components was just following the tutorial. I didn't realize it was outdated😅 
Yes, I'd appreciate an explanation about react hooks!